### DISCUSSION TITLE
Changing Rofi Image

### DISCUSSION THREAD

**Discussion by truensado** [prasanthrangan/hyprdots#1567] [Q&A]
I'm trying to change the rofi background image to be something other than the background screen. In the past I could just find the image.rofi and just replace it with the image of my choice but now that doesn't seem to be an option.

**Comment by kRHYME7** [comment]
You can navigate to each of the .rasi files. See image on which line holds the "wall" which is the image used.

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/61d1e066-54a0-4e60-98b5-1ada466ed883)

**Reply by truensado** [reply]
Worked out perfectly. Thank you so much for your help.
